ui/app/AppLayouts/Communities/views/OwnerTokenWelcomeView.qml

```diff
@@ -0,0 +1,198 @@
+
+    padding: 0
+    contentWidth: mainLayout.width
+    contentHeight: mainLayout.height
+
+    QtObject {
+        id: d
+
+        function generateSymbol(isOwner) {
+            // TODO: Add a kind of verification for not repeating symbols
+            const shortName = root.communityName.substring(0, 3)
+            if(isOwner)
+                return "OWN" + shortName.toUpperCase()
+            else
+                return "TM" + shortName.toUpperCase()
+        }
+    }
+
+    ColumnLayout {
+        id: mainLayout
+
+        width: root.viewWidth
+        spacing: 20
+
+        StatusBaseText {
+            id: introPanel
+
+            Layout.fillWidth: true
+
+            wrapMode: Text.WordWrap
+            lineHeight: 1.2
+            font.pixelSize: 15
+            text: qsTr("Your <b>Owner token</b> will give you permissions to access the token management features for your community. This token is very important - only one will ever exist, and if this token gets lost then access to the permissions it enables for your community will be lost forever as well.<br><br>
+                        Minting your Owner token also automatically mints your community’s <b>TokenMaster token</b>.  You can airdrop your community’s TokenMaster token to anybody you wish to grant both Admin permissions and permission to access your community’s token management functions to.<br><br>
+                        Only the hodler of the Owner token can airdrop TokenMaster tokens. TokenMaster tokens are soulbound (meaning they can’t be transferred), and you (the hodler of the Owner token) can remotely destruct a TokenMaster token at any time, to revoke TokenMaster permissions from any individual.")
+        }
+
+        InfoPanel {
+            isOwner: true
+            checkersModel: [
+                qsTr("Only 1 will ever exist"),
+                qsTr("Hodler is the owner of the Community"),
+                qsTr("Ability to airdrop / destroy TokenMaster token"),
+                qsTr("Ability to mint and airdrop Community tokens")
+            ]
+        }
+
+        InfoPanel {
+            isOwner: false
+            showTag: true
+            checkersModel: [
+                qsTr("Unlimited supply"),
+                qsTr("Grants full Community admin rights"),
+                qsTr("Ability to mint and airdrop Community tokens"),
+                qsTr("Non-transferrable"),
+                qsTr("Remotely destructible by the Owner token hodler")
+            ]
+        }
+
+        component InfoPanel : Rectangle {
+            id: panel
+
+            property bool isOwner
+            property bool showTag
+            property alias checkersModel: checkersItems.model
+            readonly property int margins: 24
+
+            Layout.fillWidth:  true
+            Layout.preferredHeight: panelRow.implicitHeight
+
+            color: "transparent"
+            radius: 8
+            border.color: Theme.palette.baseColor2
+            clip: true
```

Comment:
Is clipping needed? I guess if we're going to need clipping, there's a bug somewhere and the content is overflowing. Right?

Reply:
Leftover!!! Removed!!